### PR TITLE
Update README.md

### PR DESCRIPTION
**Pull request by mohith-varma01**

### CODE REVIEW

**Comment by sanvake** [comment]
Has everything that the question asked for. **GGs**
Try doing it all in a single small breadboard, feel like you guys arent really using the breadboard properly.
Also, the wire management is neat.
Maybe try color-coding them next time?
otherwise, everything else is good.

File: README.md [hunk 1]
```diff
@@ -2,3 +2,4 @@ Mohith Varma V S
 ECE 2nd sem
 
 Day1 : https://www.tinkercad.com/things/jcDbVSAVNsk-door-bell/editel?sharecode=IZiivuxdt4T_fKD4emz4cik6u0hDy11CeMV1QPOjJCE
+Day2 : https://www.tinkercad.com/things/jcDbVSAVNsk-door-bell-with-pw/editel?sharecode=IZiivuxdt4T_fKD4emz4cik6u0hDy11CeMV1QPOjJCE
```
Comment (SunkeerthM) [inline README.md:5]: Good work. Did what was asked for

**Review by SunkeerthM — APPROVED** [review]

**Comment by Naveen-electronicenthusiast** [comment]
Looks good to me!

**Comment by VarshaSBhat** [comment]
Fulfills all the required criteria. Good job!

**Pull request merged**